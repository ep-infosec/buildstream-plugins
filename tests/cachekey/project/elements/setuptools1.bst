kind: setuptools
sources:
- kind: local
  path: files/local
