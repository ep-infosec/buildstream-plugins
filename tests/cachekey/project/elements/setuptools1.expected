90aab86db2fcc92a7200dce497fe52dce9e96de04ddbb93b04ee065ac9f708ef